### ISSUE TITLE
Error compiling flex-generated files

### ISSUE THREAD

**Issue by rmattes** [keinstein/mutabor#2]
The flex-generated files checked into git do not compile on Debian/Testing. Deleting these files will retrigger their generation and compilation will succeed.
- There seems to be  a bug in the Makefile - just touching the lexer input does _not_ trigger 
  recmpilation (which it should).
- I don't think generated files should be put under version control

**Comment by keinstein** [comment]
Thank you, for your reports. The windows build always depended on the prepared flex/bison sources. Currently, I don't know whether it is now easy to integrate both with MicroSoft Visual C++. So I decided to keep them in the source tree. But recent versions do not copy everything that is needed into the source tree. So at least the C++ sources are not self contained any more :-(

Let's keep this problem open for a while. Maybe someone comes up with a good solution.

Recompilation should at least be triggered by deleting the main C/C++ sources and headers of the lexxer and the parser.

**Comment by keinstein** [comment]
Recently, I finished creating a MingW cross compilation environment for my Debian machines. So the problem with VC is currently not so pressing.

With commit 19ef8391fe00acf788107b354237e71438a66099 I removed all generated files from git.

Actually there is still FlexLexer.h somewhere in the tree. I currently have no idea, how to deal with it. In Debian it is not distributed with flex but with libfl. And it is not automatically copied. This gets problematic as the cross compilation environment is based on arch linux packages which may be available in other versions than the system flex, which may result in incompatible versions.
